src/templates/dotnet/getDotnetVerifiedTemplateIds.ts

```diff
@@ -16,7 +16,9 @@ export function getDotnetVerifiedTemplateIds(version: string): RegExp[] {
         'ServiceBusTopicTrigger',
         'ServiceBusQueueTrigger',
         'CosmosDBTrigger',
-        'EventGridTrigger'
+        'EventGridTrigger',
+        'SQLInputBinding',
+        'SQLOutputBinding'
```

Comment:
Nit, for one less line-change in the future:
```suggestion
        'SQLOutputBinding',
```